docs/tutorial/connecting.rst

```diff
@@ -46,7 +57,13 @@ When the :code:`with`-block is exited, the :code:`shutdown` method of the instru
 Using adapters
 ==============
 
-PyMeasure supports a number of adapters, which are responsible for communicating with the underlying hardware. In the example above, we passed the string "GPIB::4" when constructing the instrument. By default this constructs a VISAAdapter class to connect to the instrument using VISA. Instead of passing a string, we could equally pass an adapter object. ::
+PyMeasure supports a number of adapters, which are responsible for communicating with the underlying hardware.
+In the example above, we passed the string "GPIB::4" when constructing the instrument.
+By default this constructs a VISAAdapter (our most popular, default adapter) to connect to the instrument using VISA.
+Passing a string (or integer in case of GPIB) is by far the most typical way to create pymeasure instruments.
+
+Sometimes, you might need to go beyond the usual setup, which is also possible.
+Instead of passing a string, you could equally pass an adapter object. ::
```

Comment:
Comment not related to this line, I add here because is the closest point where I can add it.

In my opinion, all discussion below related to `Lakeshore425` should be removed, since we no longer use this approach.
After the adapters restructuring done by @bmoneke, the only use case for using a custom made adapter is when instrument is using a phy layer protocol not currently supported by pyvisa, e.g. PCIexpress or SPI.

Reply:
Done in 0cfc003.